### PR TITLE
task_52739_subject_search

### PR DESCRIPTION
**Pull request by MinhHieu2804**
## Related Tickets
- [##52739](https://edu-redmine.sun-asterisk.vn/issues/52739)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- Dùng gem ransack serch subject và question

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by TMHuy01-08** [comment]
approved

**Comment by MinhHieu2804** [comment]
ready

File: app/views/admin/questions/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <% provide(:title, "All question") %>
 <h1>All question</h1>
 <div class="row">
+  <%= search_form_for @search,url: @search_path, class: "form-inline" do |search| %>
+    <%= search.search_field :question_content, class: "form-control" %>
```
Comment (manhnt-0960) [inline app/views/admin/questions/index.html.erb:5]: thêm order nữa e

**Comment by MinhHieu2804** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**